Libs/Widgets/ctkConsole.cpp

```diff
@@ -94,7 +95,10 @@ ctkConsolePrivate::ctkConsolePrivate(ctkConsole& object) :
   InteractivePosition(documentEnd()),
   MultilineStatement(false), Ps1("$ "), Ps2("> "),
   EditorHints(ctkConsole::AutomaticIndentation | ctkConsole::RemoveTrailingSpaces),
-  ScrollbarAtBottom(false)
+  ScrollbarAtBottom(false),
+  RunFileOptions(ctkConsole::RunFileButton | ctkConsole::RunFileShortcut),
```

Comment:
By default, let's do:

```
RunFileOptions(ctkConsole::RunFileShortcut)
```

That way the default behaviour  is similar before and after your change.